tests/base.py

```diff
@@ -389,3 +392,10 @@ def dt_to_ts(self, dtime):
                 return date_stripped
             except ValueError:
                 continue
+    
+    def get_bookmark_formatted(self, date):
+        for _format in self.DATETIME_FMT:
+            try:
+                return dt.strptime(date, _format).strftime(self.BOOKMARK_FORMAT)
+            except ValueError:
+                continue
```

Comment:
The `dt_to_ts()` is already updated with `format` argument in tap-tester test related branch so no need to define this function. Remove this after back-merging this branch with the base branch.

Reply:
Updated the code